dali/daliadmin/daadmin.hpp

```diff
@@ -23,13 +23,17 @@
 
 #include "dautils.hpp"
 
+namespace daadmin
+{
+
 extern DALIADMIN_API void setDaliConnectTimeoutMs(unsigned timeoutMs);
 extern DALIADMIN_API void xmlSize(const char *filename, double pc);
 extern DALIADMIN_API void translateToXpath(const char *logicalfile, DfsXmlBranchKind tailType = DXB_File);
 
 extern DALIADMIN_API void _export_(const char *path, const char *dst, bool safe = false);
-extern DALIADMIN_API void import(const char *path, const char *src, bool add);
-extern DALIADMIN_API void _delete_(const char *path, bool backup);
+extern DALIADMIN_API bool importFromFile(const char *path, const char *filename, bool add, StringBuffer &out);
+extern DALIADMIN_API bool importFromXML(const char *path, const char *xml, bool add, StringBuffer &out);
+extern DALIADMIN_API bool delete(const char *path, bool backup,StringBuffer &out);
```

Comment:
I think 'delete' is causing problems for some version of the compiler, causing the failures seen in the AWS smoketest.
@wangkx - perhaps rename to 'erase'

Reply:
ok